src/java.base/share/classes/java/lang/AbstractStringBuilder.java

```diff
@@ -1706,141 +1868,201 @@ final boolean isLatin1() {
         return COMPACT_STRINGS && coder == LATIN1;
```

Comment:
```suggestion
return isLatin1(coder);
```
Can we reuse static isLatin1 like this?